src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -283,8 +282,11 @@ let countDistinct
     byAid
     |> List.choose id
     |> anonymizedSum
-    |> Option.defaultValue 0.
-    |> (Math.roundAwayFromZero >> int64 >> (+) safeCount >> max 0L >> Result.Count)
+    |> Math.roundAwayFromZero
+    |> int64
```

Comment:
I was wondering if I could move these 2 lines to `anonymizedSum`, but doesn't the latter also cater for float results, like, coming from a `SUM` aggregator?

Reply:
also, I now realize that the combination of `()`s and `>>`s was here to force it to a single line, pls confirm I'll undo this change

Reply:
> the combination of ()s and >>s was here to force it to a single line

Could be. Go with whatever you think reads better.